packages/manager/modules/pci/src/projects/new/classes/cart.class.js

```diff
@@ -31,4 +39,11 @@ export default class PciCartProject {
     this.items.push(item);
     return item;
   }
+
+  deleteItem(itemOptions) {
+    const itemIndex = this.items.findIndex(
```

Comment:
you can also use https://lodash.com/docs/4.17.15#remove which might be cleaner

Reply:
As discuss together i let it as now to avoid usage of external deps.